### PR TITLE
Update CommonStore.java

### PR DESCRIPTION
**Pull request by 451PCBCOM**
Delete unused import.

**What does this PR do?**

**Why are these changes required?**

**This PR has been tested by:**
- Unit Tests
- Manual Testing

**Follow up**

**Extra details**

### CODE REVIEW

File: chainbase/src/main/java/org/tron/core/db/CommonStore.java [hunk 1]
```diff
@@ -34,4 +33,9 @@ public BytesCapsule get(byte[] key) {
   public boolean has(byte[] key) {
     return dbSource.getData(key) != null;
   }
+
+  @Override
+  public String getDbName() {
+    return null;
```
Comment (shydesky) [inline chainbase/src/main/java/org/tron/core/db/CommonStore.java]: this is a wrong implement.

Comment (451PCBCOM) [reply chainbase/src/main/java/org/tron/core/db/CommonStore.java]: yes, sorry. 
This is unnecessary to add:

  @Override
  public String getDbName() {
    return null;

File: chainbase/src/main/java/org/tron/core/db/CommonStore.java [hunk 2]
```diff
@@ -34,4 +33,9 @@ public BytesCapsule get(byte[] key) {
   public boolean has(byte[] key) {
     return dbSource.getData(key) != null;
   }
+
```
Comment (451PCBCOM) [inline chainbase/src/main/java/org/tron/core/db/CommonStore.java]: yes, sorry.
This is unnecessary to add.

File: chainbase/src/main/java/org/tron/core/db/CommonStore.java [hunk 3]
```diff
@@ -34,4 +33,9 @@ public BytesCapsule get(byte[] key) {
...
+  @Override
```
Comment (451PCBCOM) [inline chainbase/src/main/java/org/tron/core/db/CommonStore.java]: yes, sorry.
This is unnecessary to add:

File: chainbase/src/main/java/org/tron/core/db/CommonStore.java [hunk 4]
```diff
@@ -34,4 +33,9 @@ public BytesCapsule get(byte[] key) {
...
+  public String getDbName() {
```
Comment (451PCBCOM) [inline chainbase/src/main/java/org/tron/core/db/CommonStore.java]: yes, sorry.
This is unnecessary to add:

File: chainbase/src/main/java/org/tron/core/db/CommonStore.java (hunk 1 above)
Comment (451PCBCOM) [inline chainbase/src/main/java/org/tron/core/db/CommonStore.java]: yes, sorry.
This is unnecessary to add:

**Pull request closed without merging**